File: single-language-analysis/dataset-files/java/TestSmell060.java
// Original URL: https://github.com/elsabnilsson/lab1/blob/6cb40dfe2873d0912b7c307f2f873c3689f6a802/CarTest.java#L125-L132

public class TestSmell060 {
    @Test
    void decrementSpeed() {
        Volvo240 volvo240 = new Volvo240(0,0);
        volvo240.startEngine();
        volvo240.decrementSpeed(0.01);
        double delta = 0.0001;
        assertEquals(0.0875, volvo240.getCurrentSpeed(), delta);
    }
}

File: single-language-analysis/dataset-files/python/test_smell_060.py
import pytest

class TestCarTest:
    def test_smell_060(self):
        volvo240 = Volvo240(0, 0)
        volvo240.start_engine()
        volvo240.decrement_speed(0.01)
        delta = 0.0001
        assert 0.0875 == pytest.approx(volvo240.get_current_speed(), rel=delta)